### ISSUE TITLE
problems with bitmerchant

### ISSUE THREAD

**Issue by valsha** [dessalines/bitmerchant#10]
Problems with runing bitmerchant.
Just run java -jar bitmerchant-shaded.jar (downloaded from link) -loglevel INFO login to http://localip:4567
in Wallet Settings i have only:
1. unlock wallet
2, remove password
in console i can see it 
12:13:27.724 [NioClientManager] INFO  org.bitcoinj.core.Peer - Received alert from peer [2001:b]:8333: URGENT: Alert key compromised, upgrade required
Button Generator is not working, 

I can see warning messages in admin cp, but all is blank.

**Comment by valsha** [comment]
just try building from scratch...no luck, all the same problems.

**Comment by dessalines** [comment]
Same as the other, this would require an update to bitcoinj to fix, and I've since focused on other projects. If there's enough interest I can start working on this again.

**Comment by valsha** [comment]
Can it be a paid job? :)

**Comment by dessalines** [comment]
No, it would require too much time. Someone can submit a PR though.
